.ci/install_kata_kernel.sh

```diff
@@ -18,6 +18,9 @@ cidir=$(dirname "$0")
 source "${cidir}/lib.sh"
 source "/etc/os-release" || source "/usr/lib/os-release"
 
+latest_build_url="http://jenkins.katacontainers.io/job/kernel-nightly-$(uname -m)/lastSuccessfulBuild/artifact/artifacts"
```

Comment:
I don't know if [kata-arch.sh](https://github.com/kata-containers/tests/blob/master/.ci/kata-arch.sh) would be a better thing than `uname` here?

Reply:
I dont want to depend a lot of the script, I think uname is fine for most of the uses, I think kata-arch is great for get right paths to golang. @jodh-intel  any preference ?

Reply:
Well, I think we were hoping that `kata-arch.sh` would be the "Mother of all arch commands to end all arch commands" but if it doesn't buy you anything I guess stick with `uname`.